apps/prairielearn/src/schemas/schemas/questionOptionsMultipleTrueFalse.json

```diff
@@ -19,7 +19,8 @@
       "type": "array",
       "items": {
         "type": "string"
-      }
+      },
+      "minItems": 1
```

Comment:
Why this change? AFAICT there's no reason one coudln't have a question with zero true statements.

Reply:
I took a brief look at `apps/prairielearn/public/localscripts/calculationQuestion/MTFServer.js` and couldn't see anything that relies on there being 0 true statements. That being said, should we enforce this?

Reply:
My default is to change absolutely nothing about v2 questions. If we can prove this is completely safe, let's land this in a separate PR first.